script/Arcueid/Day 8/08_00_ARC08_6B.txt

```diff
@@ -742,7 +742,7 @@ No problem at all. If I can't dodge it, then I'll just have to tear it apart in
 [sha:6ff94c5a0b3a04f0e6f92a4c683212948fa9048f]{
 -- Page 400, Offset 20694.
 -- 逃げる背中を追う。<赤狂|あかぐる>う死の断線。
-// I have no idea what this second line means
+// I have no idea what this second line means me neither
```

Comment:
I'm not gonna sit here and pretend like I do, but just grammatically the current iteration is mixing the sentence all up:
> The disconnection of those crazy red lines of death.